core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1179,12 +1179,12 @@ public CommitId flush(boolean force, boolean waitIfOngoing) throws EngineExcepti
                     try {
                         translog.prepareCommit();
                         logger.trace("starting commit for flush; commitTranslog=true");
-                        commitIndexWriter(indexWriter, translog, null);
+                        final long committedGeneration = commitIndexWriter(indexWriter, translog, null);
```

Comment:
I don't understand this comment, this is a new local variable that was never there before. Did you maybe mean to comment on `Translog#lastCommittedTranslogFileGeneration` which we have discussed the confusing nature of before?